string train_command
string train_name
int8 train_number
int8 speed_value
int64 accessories_value
